Client/Frontend/Browser/Playlist/Controllers/PlaylistViewController.swift

```diff
@@ -1102,3 +1135,9 @@ extension PlaylistViewController: VideoViewDelegate {
     return true
   }
 }
+
+extension PlaylistFolder {
+  var isPersistent: Bool {
+    managedObjectContext?.persistentStoreCoordinator?.persistentStores.first(where: { $0.type == "InMemory" }) == nil
```

Comment:
Does "InMemory" match `NSPersistentStore.StoreType.inMemory.rawValue`? Maybe we can use that instead of hardcoding

Reply:
Cool, we can remove it whenever we go 15+ and do a bulk removal around 15.0 checks